src/bin/main.rs

```diff
@@ -41,6 +46,8 @@ fn core() -> Result<i32, String> {
             writeln!(stdout, "You are currently running Bonnie v{}! You can see the latest release at https://github.com/arctic-hen7/bonnie/releases.", BONNIE_VERSION).expect("Failed to write version.");
             return Ok(0);
         } else if prog_args[0] == "-i" || prog_args[0] == "--init" {
+            print!("Initialising...");
+
```

Comment:
Nice touch! We are however just copying from a file, so maybe just the second part!